packages/loot-core/src/server/db/index.ts

```diff
@@ -175,7 +197,10 @@ export async function select(table, id) {
     [id],
     true,
   );
-  return rows[0];
+  // TODO: In the next phase, we will make this function generic
+  // and pass the type of the return type to `runQuery`.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return rows[0] as any;
```

Comment:
This will be made generic on the the next PRs so that we can control the PR size